### ISSUE TITLE
Demo of accessibility features

### ISSUE THREAD

**Issue by tomcrane** [digirati-co-uk/iiif-canvas-panel#144]
Documentation page:

https://deploy-preview-137--iiif-canvas-panel.netlify.app/docs/examples/accessibility

It won't be exactly like that draft, but what we do have should be wrapped up in a demo that can be tested with assistive technologies.
